packages/otp2-tile-overlay/src/index.tsx

```diff
@@ -56,6 +58,11 @@ const OTP2TileLayerWithPopup = ({
    * in accordance with the MapLibreGL api.
    */
   onMapClick?: (event: EventData) => void
+  /**
+   * A method to request the agency for a given stop ID. If this method is not passed, the agency
+   * for a given stop ID will not be displayed in the popup.
+   */
+  requestAgencyForStop?: (stopId: string) => void
```

Comment:
Why do we have this as well as the map. Would it be possible to just have this instead? Feels cleaner

Reply:
Can we get rid of this line then? The method is never defined

Reply:
It's defined in the OTP-RR PR. https://github.com/opentripplanner/otp-react-redux/pull/1433/files